tools/run_tests/performance/loadtest_config.py

```diff
@@ -378,6 +417,19 @@ def main() -> None:
                       '--output',
                       type=str,
                       help='Output file name. Output to stdout if not set.')
+    argp.add_argument(
+        '-offered_loads_list',
+        nargs="*",
+        type=int,
+        default=[],
+        help='A list of offered loads that the tests are running with.')
+    argp.add_argument('-client_channel',
+                      type=int,
+                      help='Number of client channel opened for the test')
+    argp.add_argument(
```

Comment:
I would call the argument '--server_threads' and in the description write 'number of async server threads'.

I would also list the arguments in the logical order: client_channels, server_threads, offered_loads.